bundle.Dockerfile

```diff
@@ -40,11 +40,6 @@ LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.32.0
 LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
```

Comment:
Consider mentioning in the commit message the commit this was introduced. 
https://github.com/openshift/windows-machine-config-operator/pull/1965/commits/c91335a5eea3f025d2e99a73b1a3b216dddb15ad